docs/en/analytics/data-mgmt/metadata-mgmt.md

```diff
@@ -0,0 +1,68 @@
+# Metadata management
+The solution automatically scans clickstream data to generate metadata and then stored them in Redshift on a daily basis. There are three type of metadata:
+
+1. **Event**: metadata describes clickstream events, which are stored in `event_metadata` table in Redshift.
+2. **Event Parameter**: metadata describes clickstream event parameters, which are stored in `event_parameter_metadata` table in Redshift.
```

Comment:
It's a temporary table for one day only. The historical results persist in DDB. @dengmingtong , help to confirm it.